### PR TITLE
Fix touch issue when dragging a widget with SVG icon

### LINKED ISSUE #687: Widget disappears while dragging . Occurs in iPad, Chrome tablet mode
Widget disappears while dragging . Occurs in iPad, Chrome tablet mode

Steps to Reproduce:
Float two or more widgets
Reduce the width of widget to such an extent where only widget icon displays in the title
Select a tab
Drag the non-selected tab --> One of the widget disappears.

Environment:
User said it occurs in their site and My iTwins as well. It occurs only in iPad or Chrome table mode. Not occurring in browser with full screen. It occurs only after updating iTwin.js 4.x
Refer to this video for reproduction steps:
https://github.com/iTwin/viewer-components-react/assets/107229931/ccdf7811-f0a5-4c8a-9988-7f2eef3df864

### PR DESCRIPTION
**Pull request by GerardasB**
## Changes

This PR fixes an issue with incorrect event handling of widget tabs when SVG icon is targeted by a touch device.

Fixes #687 

## Testing

Tested manually via a standalone test-app.

### CODE REVIEW

**Comment by GerardasB** [comment]
@Mergifyio backport release/4.8.x

File: test-apps/appui-test-app/appui-test-providers/src/ui/providers/WidgetApiStageUiItemsProvider.tsx [hunk 1]
```diff
@@ -135,7 +143,7 @@ export class WidgetApiStageUiItemsProvider implements UiItemsProvider {
       widgets.push({
         id: "WR-A",
         label: "WR-A",
-        icon: "icon-text-align-text-align-left",
+        icon: <SvgTextAlignLeft />,
```
Comment (raplemie) [inline test-apps/appui-test-app/appui-test-providers/src/ui/providers/WidgetApiStageUiItemsProvider.tsx:146]: Does it works correctly for webicons ? Or only for SVG icons ?

**Review by raplemie — APPROVED** [review]
LGTM.

File: test-apps/appui-test-app/appui-test-providers/src/ui/providers/WidgetApiStageUiItemsProvider.tsx (hunk 1 above)
Comment (GerardasB) [reply test-apps/appui-test-app/appui-test-providers/src/ui/providers/WidgetApiStageUiItemsProvider.tsx:146]: It worked correctly with webicons.

**Pull request merged**